pkg/controller/idler/idler_controller.go

```diff
@@ -165,9 +169,11 @@ func (r *ReconcileIdler) ensureIdling(logger logr.Logger, idler *toolchainv1alph
 // and scales the owner down to zero and returns "true".
 // Otherwise returns "false".
 func (r *ReconcileIdler) scaleControllerToZero(logger logr.Logger, meta metav1.ObjectMeta) (bool, error) {
+	logger.Info("Scaling controller to zero", "name", meta.Name)
 	owners := meta.GetOwnerReferences()
 	for _, owner := range owners {
 		if owner.Controller != nil && *owner.Controller {
+			log.Info("owner", "owner", owner.Kind)
```

Comment:
Should we change this to `kind` for the label?
```suggestion
			log.Info("owner", "kind", owner.Kind)
```

Reply:
Yeah.. the additional logging I added was mostly for debugging. We need to clean it up a bit.